sklearn/tests/test_common.py

```diff
@@ -385,3 +389,25 @@ def test_transformer_n_iter():
 
         if hasattr(estimator, "max_iter") and name not in external_solver:
             yield check_transformer_n_iter, name, estimator
+
+
+def test_partial_fit():
+    """Tests all estimators which support partial_fit"""
+    
+    skiplist = ("HashingVectorizer",
+                "MiniBatchDictionaryLearning",
+                "LSHForest")
+
+    for name, Estimator in all_estimators():
+        if (not hasattr(Estimator, 'partial_fit')) or (name in skiplist):
+            continue
+        # General tests for partial_fit method of estimators
+        yield check_estimators_pfit_general, name, Estimator
+        # Test if change in features in 2nd call raises ValueError
+        yield check_estimators_pfit_n_features, name, Estimator
+        # Test if fit after multiple partial_fits resets the estimator
+        yield check_estimators_pfit_reset_when_fit, name, Estimator
+        # Check the correct parsing of the classes argument
+        yield check_estimators_pfit_classes_arg, name, Estimator
+        # Tests related to the effect of partial fit tests
+        #yield check_estimators_pfit_effect, name, Estimator
```

Comment:
@jnothman 
Have commented this out since it doesn't pass for 7 classifiers/regressors ( SGDClassifier / SGDRegressor / MultinomialNB, GaussianNB, etc... ) please advice on how I should proceed with this test alone...